fix/empty-wheelbarrows.lua

```diff
@@ -0,0 +1,21 @@
+--checks all wheelbarrows on map for rocks stuck in them. If a wheelbarrow isn't in use for a job (hauling) then there should be no rocks in them
+--rocks will occasionally get stuck in wheelbarrows, and accumulate if the wheelbarrow gets used.
+--this script empties all wheelbarrows which have rocks stuck in them.
+
+
+for _,e in ipairs(df.global.world.items.other.TOOL) do
+    -- wheelbarrow must be on ground and not in a job
+    if ((not e.flags.in_job) and e.flags.on_ground) then
+        if e.subtype.id == "ITEM_TOOL_WHEELBARROW" then
```

Comment:
using the `isWheelbarrow()` item vmethod for this test would be more resistant to someone who changes the raws

technically any tool with the `HEAVY_OBJECT_HAULING` tool use flag is a "wheelbarrow"

Reply:
Will adjust to use `e:isWheelbarrow()`